backstop_data/engine_scripts/puppet/price-set/add-price-fields.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Page = require('../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, viewport) => {
+  const page = new Page(engine, scenario, viewport);
+  const pricelistEmpty = await engine.$('#crm-main-content-wrapper .messages');
+
+  if(pricelistEmpty) {
```

Comment:
There should be a space after `if`. Again, not sure if the linter is enabled.

Reply:
Fixed!